src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,43 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private final int maxSize = 10;
```

Comment:
```suggestion
    private static final int MAX_SIZE = 10;
```
this should be a constant

Reply:
a cant pass tests if I put here capital letters